### ISSUE TITLE
twisted.enterprise.adbapi has a lot of simple twistedchecker errors

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8304]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8304|
|Type|enhancement|
|Created|2016-04-26 03:33:41Z|
|Branch|https://github.com/twisted/twisted/tree/adbapi-txchecker-8304|

A whole bunch of these are easily solved, like the newlines and the docstrings lacking links.

<details><summary>Searchable metadata</summary>

```
trac-id__8304 8304
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_adbapi_txchecker_8304 branches/adbapi-txchecker-8304
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1461641621873899 1461641621873899
changetime__1461810067100196 1461810067100196
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [47321]) Branching to adbapi-txchecker-8304.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Builders spun, all green when they finish going through the backlog. Compare trunk (https://buildbot.twistedmatrix.com/builders/twistedchecker/builds/4056/steps/run-twistedchecker/logs/twistedchecker%20twisted.enterprise%20errors) with this branch (https://buildbot.twistedmatrix.com/builders/twistedchecker/builds/4058/steps/run-twistedchecker/logs/twistedchecker%20twisted.enterprise%20errors).

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Looks good.

Why not also update the inline comments to use 2 spaces?


Please check my comment and merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

Hi Adi, thanks for the review.

The coding standard doesn't mention 2-spaces, and a lot of Twisted code has one space -- I have opened [#8310](https://github.com/twisted/twisted/issues/8310)#ticket so we can solve this officially, and we can revisit it in the codebase once we decide there.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [47332]) Merge adbapi-txchecker-8304: Fix some low-hanging TwistedChecker errors in twisted.enterprise.adbapi.

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8304](https://github.com/twisted/twisted/issues/8304)
